packages/celotool/src/cmds/deploy/destroy/vm-testnet.ts

```diff
@@ -0,0 +1,29 @@
+import { DestroyArgv } from '@celo/celotool/src/cmds/deploy/destroy'
+import {
+  destroyTerraformModule,
+  initTerraformModule,
+  planTerraformModule,
+} from '@celo/celotool/src/lib/terraform'
+import { confirmAction, envVar, fetchEnv } from '@celo/celotool/src/lib/utils'
+
+export const command = 'vm-testnet'
+export const describe = 'destroy an existing VM-based testnet'
+
+export const builder = {}
+
+const terraformModule = 'testnet'
+
+export const handler = async (argv: DestroyArgv) => {
+  const envType = fetchEnv(envVar.ENV_TYPE)
+  console.info(`Destroying ${argv.celoEnv} in environment ${envType}`)
+
+  console.info('Initializing...')
+  await initTerraformModule(terraformModule)
+
+  console.info('Planning...')
+  await planTerraformModule(terraformModule, true)
+
+  await confirmAction(`Are you sure you want to destroy ${argv.celoEnv} in environment ${envType}?`)
```

Comment:
We have usually only ask for confirmation in staging/production environments which may apply here as well

Reply:
I think it'd be good to have the extra amount of control for all environments especially when updating-- it shows you a plan for what exactly has to be changed which is helpful in understanding exactly what's going to happen